collector/lib/NetworkSignalHandler.cpp

```diff
@@ -173,8 +177,10 @@ std::vector<std::string> NetworkSignalHandler::GetRelevantEvents() {
         "recvmsg<",
         "recvmsg>",
     };
+
+    base_events.insert(base_events.end(), extra.begin(), extra.end());
```

Comment:
You could use an initializer list here directly.
```suggestion
    base_events.insert(base_events.end(), {
        "sendto<",
        "sendto>",
        "sendmsg<",
        "sendmsg>",
        "sendmmsg<",
        "recvfrom<",
        "recvfrom>",
        "recvmsg<",
        "recvmsg>",
        "recvmmsg<",
        "recvmmsg>",
        "recvmsg<",
        "recvmsg>",
    });
```